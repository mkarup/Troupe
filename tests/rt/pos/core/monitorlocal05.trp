let val p = spawn (fn () => receive [ hn x => () ])
    val monRef = monitorlocal (p, `{}`)
    val _ = send (p, 42)
in
  receive [ hn (str, monRef', from, reason) => str = "DONE" ]
end