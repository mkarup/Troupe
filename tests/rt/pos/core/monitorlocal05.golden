2024-12-11T12:25:44.553Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: true@{}%{}
